torch/autograd/graph.py

```diff
@@ -108,6 +108,13 @@ def register_hook(self, fn: Callable[..., Any]) -> RemovableHandle:
             See :ref:`backward-hooks-execution` for more information on how when this hook
             is executed, and how its execution is ordered relative to other hooks.
 
+        .. note::
+            In the rare case where the hook is registered while the Node has already
+            begun execution, there is no longer any guarantee on :attr:`grad_outputs` and
+            so the hook should not access the :attr:`grad_outputs` argument. The
```

Comment:
```suggestion
            begun execution, there is no longer any guarantee on :attr:`grad_outputs` content
            (it might be either as usual or empty depending on other factors). The
```